tensorflow/lite/micro/micro_utils.h

```diff
@@ -44,11 +44,11 @@ T FloatToQuantizedType(const float value, const float scale, int zero_point) {
 
 template <typename T>
 T FloatToSymmetricQuantizedType(const float value, const float scale) {
-  int32_t result = round(value / scale);
-  result =
-      std::max(static_cast<int32_t>(std::numeric_limits<T>::min() + 1), result);
-  result =
-      std::min(static_cast<int32_t>(std::numeric_limits<T>::max()), result);
+  std::int64_t result = round(value / scale);
```

Comment:
why do we need to change to 64 bits? might be worth a comment.

Reply:
Added a comment explaining that 8x16 conv requires a 64-bit bias, so numeric_limits<int64_t> must be an int64.